README.md

```diff
@@ -254,6 +254,7 @@ When learning CS there are some useful sites you must know to get always informe
 - [Unsupervised Sentiment Neuron](https://blog.openai.com/unsupervised-sentiment-neuron/) : Unsupervised Sentiment Neuron 
 - [Robots that learn](https://blog.openai.com/robots-that-learn/) : Robots that Learn 
 - [grakn.ai](https://grakn.ai) : The Database for AI 
+- [fast.ai](http://course.fast.ai/) : Extremely well done course on AI. Takes 7 weeks from zero to hero.
```

Comment:
@shinobiultra Remove the trailing slash from the URL if you want it to pass on Travis. 

Change the url from: `[fast.ai](http://course.fast.ai/)` to `[fast.ai](http://course.fast.ai)`.

```
1.15s$ awesome-lint
  257:3-257:36  error    Remove trailing slash (http://course.fast.ai)  trailing-slash
✖ 1 error
The command "awesome-lint" exited with 1.
```